src/Makevars.win

```diff
@@ -1,4 +1,4 @@
+#PKG_CPPFLAGS = -DARMA_NO_DEBUG -DBOOST_DISABLE_ASSERTS
+PKG_CXXFLAGS = -I. $(SHLIB_OPENMP_CXXFLAGS) -ftrack-macro-expansion=0
```

Comment:
One more thing I want to ask here is that -- will cran allow us to use `-ftrack-macro-expansion=0` for windows build ? Else we may face some memory issue for windows build, may be due to : https://sourceforge.net/p/mingw-w64/mailman/message/33182613/ or http://gcc.gnu.org/bugzilla/show_bug.cgi?id=56746.

Reply:
@Yashwants19 we'll need to probably communicate with CRAN in-advance. I'm not sure that it will be feasible. That said, the CRAN build servers are stronger than the GitHub Actions/Appveyor build system.

Reply:
I only found one package([ChannelAttribution](https://github.com/cran/ChannelAttribution/blob/474d7f4588c36ad0eb2e2e22d058a6b5db8173a3/src/Makevars.win))  here that use this flag, or I am again making some blunder here?

Reply:
No blunder, it looks like the check matrix is reporting everything is fine. 

https://cran.r-project.org/web/checks/check_results_ChannelAttribution.html

It may be possible to use this flag, but we should only opt-in to flags as a last resort. We're stressing this because any additional compiler flags may impact that package's portability.

Reply:
I think so Cf bindings is costing us some heavy memory usage, as by removing cf bindings, windows build is working smoothly on GH too. Hence cran's stronger server might not face this issue even with cf bindings, as it working fine locally.

Reply:
Should I remove this flag in the main repository(Yashwants19/mlpack)?

Reply:
Let's aim to remove it for now. We can bring it back later if need be.

Reply:
Done ([aff22c1](https://github.com/Yashwants19/mlpack/pull/8/commits/aff22c17920647350bca92926c8f508cad9fac06)).